backend/controllers/Events.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const eventRouter = express.Router()
+
+var mongodb = require('mongodb');
+var Event = require('../models/Event');
+
+eventRouter.use(cors())
+eventRouter.use(bodyParser.json());
+eventRouter.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+eventRouter.get('/list', function(req, res) {
+  Event.find(function(err, events) {
+		if (err) {
+			res.json(err);
+		}
+		events = events.map(q => q.toJSONFor());
```

Comment:
What do you think about renaming this to a different variable?

Reply:
Good idea, renamed in fix.